test/core/utils/get-selector.js

```diff
@@ -551,6 +551,16 @@ describe('axe.utils.getSelector', function () {
 		);
 	});
 
+	it('should work correctly when a URL attribute cannot be shortened', function () {
+		var node = document.createElement('a');
+		node.setAttribute('href', 'mars2.html?a=be_bold');
+		fixture.appendChild(node);
+		axe._tree = axe.utils.getFlattenedTree(document.documentElement);
+
+		assert.isTrue(axe.utils.getSelector(node).indexOf('mars2.html?a=be_bold') !== -1);
```

Comment:
Suggestion: `assert.notInclude(axe.utils.getSelector(node), 'mars2.html?a=be_bold');`

Reply:
you mean `assert.include`

Reply:
Oops. Yes!